runner.py

```diff
@@ -12,16 +12,6 @@
 
 from BTVNanoCommissioning.workflows import workflows
 
-# This would crash if the ExampleWorkflow does not exist
-from ExampleWorkflow.workflows import workflows
```

Comment:
Maybe it's better to put example workflow together with `CoffeaRunner`, workflows from `BTVNanoCommissioning` are depreciated now

Reply:
Maybe. But I am not sure we need the runner.py script in CoffeaRunner, since we switched to config runner. Maybe it's better to remove it in our fork and add to ignore list for merging with BTV?

Reply:
sounds good, just remove it in our fork and add to ignore list for merging with BTV